packages/hoprd/src/api/v2/paths/sign.ts

```diff
@@ -0,0 +1,47 @@
+import { Operation } from 'express-openapi'
+import { u8aToHex } from '@hoprnet/hopr-utils'
+import { encodeMessage } from '../../../commands/utils'
+
+export const parameters = []
+
+export const POST: Operation = [
+  async (req, res, _next) => {
+    const message = encodeMessage(req.body.body)
```

Comment:
Don't think this is what we want.

Assuming the following workflow:
- want to send "Hi, this is Alice"
- signs "Date.now + msg"
- call `/send` which prefixes the message with Date.now(), hence we have "Date.now() + Date.previously + msg"

Reply:
Signing the received msg is therefore sufficient.

Reply:
I see your point, prob. indeed worth removing `encodeMessage` then, good catch!

Reply:
Solved in ce79363